lib/dialects/pgnative/index.js

```diff
@@ -0,0 +1,118 @@
+// PostgreSQL Native Driver (pg-native)
+// -------
+const { Client } = require('pg');
+const Client_PG = require('../postgres');
+const { makeEscape } = require('../../util/string');
+
+class Client_PgNative extends Client_PG {
+  _stream() {
+    console.log('HERE WE GO');
```

Comment:
This will get rebased out after debugging complete.

@kibertoad since `pg-native` is incompatible with `pg-query-stream` I'll need to either:

1. Disable support for streaming as something like the below line where an exception gets thrown at invocation (though this method isn't the correct place from what I can tell, perhaps something in the execution lib)
2. Attempt to implement a stream that has the same feeling, but will not have the same response characteristics. 
2.